e2e-saas/mocha-pr-check.opts

```diff
@@ -1,5 +1,5 @@
 --timeout 120000
---reporter 'node_modules/e2e/dist/driver/CheReporter.js'
```

Comment:
why the upstream reporter works for 7.9.1 ?

Reply:
Because we weren't using any API requests in the older versions